sys/dev/athn/athn.c

```diff
@@ -852,6 +867,7 @@ athn_config_pcie(struct athn_softc *sc)
 	athn_write_serdes(sc, sc->serdes);
 
 	DELAY(1000);
+	ATHN_LOCK(sc);
```

Comment:
It shall be before `athn_write_serdes` but this function looks like PCIE related so it makes no difference because we will remove it :)

Reply:
Currently function 'athn_write_serdes' has own lock/unlock inside, I don't know if it's good solution, but is't used in many different places also.